src/routes/extra_functionality_movie.py

```
@@ -0,0 +1,139 @@
+    if existing:
+        if data.is_liked is None:
+            return MessageResponse(message="No change")
+
+        if existing.is_liked == data.is_liked:
+            await db.delete(existing)
+            await db.commit()
+            return MessageResponse(message="Reaction removed")
+        else:
+            existing.is_liked = data.is_liked
+            await db.commit()
+            return MessageResponse(message="Reaction updated")
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)